conf/messages.en

```diff
@@ -296,4 +296,11 @@ itemSealsInformation.error.invalidCharacter = Information must not include < and
 itemSealsInformation.error.character = Information must contain letters or numbers
 itemSealsInformation.checkYourAnswers.label = Information about broken seal(s)
 itemSealsInformation.checkYourAnswers.change.hidden = information about broken seal(s)
-itemSealsInformation.checkYourAnswers.addMoreInformation = Enter more information about broken seal(s) (optional)
\ No newline at end of file
+itemSealsInformation.checkYourAnswers.addMoreInformation = Enter more information about broken seal(s) (optional)
+addedItems.title.singular = You have given information for 1 item
+addedItems.title.plural = You have given information for {0} items
+addedItems.heading.singular = You have given information for 1 item
+addedItems.heading.plural = You have given information for {0} items
+addedItems.addAnother = Do you need to give information for another item?
```

Comment:
According to copydeck - this message should be **Do you want to give information for another item?**

Reply:
Will update now, think I copied from the prototype by mistake.